docs/source/_themes/sphinx_rtd_theme/static/css/theme.css

```diff
@@ -3525,6 +3525,10 @@ h6 {
   font-size: 100%;
 }
 
+.rst-content .toctree-wrapper p.caption {
```

Comment:
Unless you have other elements with class "caption" laying around, `p` is not needed here. One again, minor as it gets.